jointstate_topic: /manipulator/joint_states
queue_size: 16
command_topics:
    base_cyl: roboclaw_driver/128/motor1/set_joint_stated
    cyl_arm1: roboclaw_driver/129/motor1/set_joint_stated
    arm1_arm2: roboclaw_driver/129/motor2/set_joint_stated
    arm2_arm3: roboclaw_driver/130/motor1/set_joint_stated
    arm3_tool: roboclaw_driver/130/motor2/set_joint_stated
control_modes:
  cartesian:
    max_velocity: 0.05
    acceleration: 0.01
    max_error: 0.001
    interpolation_rate: 10.0
    velocity_weighing: [1, 1, 1, 0.01, 0.7]
  jointspace:
    max_velocity: 0.25
    acceleration: 0.03
    max_error: 0.001
    interpolation_rate: 10.0
    velocity_weighing: [1, 1, 1, 1, 0.05]
  incremental:
    send_rate: 10.0

